comid/measurement.go

```diff
@@ -51,6 +57,15 @@ func (o Mkey) GetPSARefValID() (PSARefValID, error) {
 	}
 }
 
+func (o Mkey) GetKeyUint() (uint64, error) {
+	switch t := o.val.(type) {
+	case uint64:
+		return t, nil
+	default:
+		return 0, fmt.Errorf("measurement-key type is: %T", t)
```

Comment:
Not that it really matters here, but only as a question of style I'd return `MAX_UINT64` rather than 0 here.